clients/go/cmd/zbctl/main_test.go

```diff
@@ -95,10 +95,22 @@ var tests = []struct {
 		goldenFile: "testdata/create_worker.golden",
 	},
 	{
-		name:       "activate job",
+		name:       "empty activate job",
+		cmd:        "--insecure activate jobs jobType --maxJobsToActivate 0",
+		goldenFile: "testdata/empty_activate_job.golden",
+	},
+	{
+		name:       "single activate job",
 		setupCmds:  []string{"--insecure deploy testdata/job_model.bpmn", "--insecure create instance jobProcess"},
-		cmd:        "--insecure activate jobs jobType",
-		goldenFile: "testdata/activate_job.golden",
+		cmd:        "--insecure activate jobs jobType --maxJobsToActivate 1",
+		goldenFile: "testdata/single_activate_job.golden",
+	},
+	{
+		name: "double activate job",
+		// we deploy on the end again to spent more time in setup phase to avoid a race condition, that we can activate more jobs then one
```

Comment:
This isn't obvious to me, can you elaborate? By deploying this means we spend more time in setup, so we don't have to wait for the jobs to be created?

Reply:
If that's correct, then wouldn't setting a longer time out on activate jobs work? Or is that not possible via zbctl?

Reply:
Ok I tried with requestTimeout 30s, but doesn't work. The thing is that only one job exist when the activate command is send (without the second deployment). This job is immediately returned. In general this is fine because this is just the max value we are setting, but in this test we want explicitly 2 because we want to test the output. This is the reason why I added the second deployment to have more time such that the second job is available. Sleep would also be ok but I dont know where to put this, since the tests are highly parameterized.

Reply:
I see what you mean - there's no way for us to wait until both jobs are created before actually activating them. My main worry is this has potential to be flaky, but I wouldn't block this for now.